CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.7.0 (December 22, 2021)
```

Comment:
This doesn't need a version bump because there's nothing new to publish. It's just an internal change.